docker/from-packages/scripts/start-collabora-online.sh

```diff
@@ -33,12 +33,22 @@ if test "${SAL_LOG-set}" == set; then
 SAL_LOG="-INFO-WARN"
 fi
 
-# Replace trusted host and set admin username and password
-perl -pi -e "s/localhost<\/host>/${domain}<\/host>/g" /etc/loolwsd/loolwsd.xml
-perl -pi -e "s/<username (.*)>.*<\/username>/<username \1>${username}<\/username>/" /etc/loolwsd/loolwsd.xml
-perl -pi -e "s/<password (.*)>.*<\/password>/<password \1>${password}<\/password>/" /etc/loolwsd/loolwsd.xml
-perl -pi -e "s/<server_name (.*)>.*<\/server_name>/<server_name \1>${server_name}<\/server_name>/" /etc/loolwsd/loolwsd.xml
-perl -pi -e "s/<allowed_languages (.*)>.*<\/allowed_languages>/<allowed_languages \1>${dictionaries:-de_DE en_GB en_US es_ES fr_FR it nl pt_BR pt_PT ru}<\/allowed_languages>/" /etc/loolwsd/loolwsd.xml
+# Replace trusted host and set admin username and password - only if they are set
+if test -n "${domain}"; then
+    perl -pi -e "s/localhost<\/host>/${domain}<\/host>/g" /etc/loolwsd/loolwsd.xml
+fi
+if test -n "${username}"; then
+    perl -pi -e "s/<username (.*)>.*<\/username>/<username \1>${username}<\/username>/" /etc/loolwsd/loolwsd.xml
+fi
+if test -n "${password}"; then
+    perl -pi -e "s/<password (.*)>.*<\/password>/<password \1>${password}<\/password>/" /etc/loolwsd/loolwsd.xml
+fi
+if test -n "${server_name}"; then
+    perl -pi -e "s/<server_name (.*)>.*<\/server_name>/<server_name \1>${server_name}<\/server_name>/" /etc/loolwsd/loolwsd.xml
+fi
+if test -n "${allowed_languages}"; then
```

Comment:
This should be `${dictionaries}`

Reply:
great catch - brain fade: thanks ! =) will merge and back-port to co-6-4 too ..